addons/registry/2.7.1/tmpl-deployment-objectstore.yaml

```diff
@@ -36,7 +36,7 @@ data:
         disable: true
       s3:
         region: "us-east-1"
-        regionendpoint: http://$OBJECT_STORE_CLUSTER_IP
+        regionendpoint: http://$OBJECT_STORE_CLUSTER_IP_BRACKETED
```

Comment:
Won't this break now with Minio or older versions of Rook?

Reply:
yeah, good catch